docs/index.asciidoc

```diff
@@ -45,6 +48,9 @@ filter {
   }
 }
 
+NOTE: Defining methods in <<plugins-{type}s-{plugin}-code>> can significantly reduce throughput. Use <<plugins-{type}s-{plugin}-init>> instead.
+
+[id="plugins-{type}s-{plugin}-using-script-file"]
 ===== Using a Ruby script file
 
 As the inline code can become complex and hard to structure inside of a text string in `code`, it's then preferrable to place the Ruby code in a .rb file, using the `path` option.
```

Comment:
```suggestion
As the inline code can become complex and hard to structure inside of a text string in `code`, it's then preferable to place the Ruby code in a .rb file, using the `path` option.
```

Reply:
This happens without the tag: 
<img width="374" alt="Screen Shot 2020-09-30 at 6 47 17 PM" src="https://user-images.githubusercontent.com/35154725/94749002-b8586580-0350-11eb-8cd1-c3d4fe79d845.png">